suites/jammy.toml

```diff
@@ -44,3 +44,17 @@ version = "1.19.1.2701-6327e27bf"
     [[direct.urls]]
     url = "https://downloads.plex.tv/plex-media-server-new/1.19.1.2701-6327e27bf/debian/plexmediaserver_1.19.1.2701-6327e27bf_amd64.deb"
     checksum = "aedee3bf69fc60fd86ca32555ce67ed6bc350d5552fd52df389d36c0b5875155"
+
+[[direct]]
+name = "vivaldi-stable"
+version = "5.6.2867.58-1"
+
+    [[direct.urls]]
+    url = "https://downloads.vivaldi.com/stable/vivaldi-stable_5.6.2867.58-1_amd64.deb"
```

Comment:
These URLs should have a ${version} placeholder so they don't need to be updated when the `version =` field above is updated.

Reply:
Fixed, thanks for noticing that!